reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -319,7 +319,7 @@
         "IpProtocol": "tcp",
         "FromPort": "10250",
         "ToPort": "10250",
-        "SourceSecurityGroupId": { "Fn::GetAtt": [ "MasterSG", "GroupId" ] }
+        "SourceSecurityGroupId": { "Fn::GetAtt": [ "NodeSG", "GroupId" ] }
```

Comment:
I wonder if there should be rules for both MasterSG and NodeSG, not all SDN solutions will require that the Master be a Node.